VideoLocker/src/main/java/org/edx/mobile/module/registration/view/IRegistrationFieldView.java

```diff
@@ -24,6 +24,11 @@
     void setActionListener(IActionListener actionListener);
 
     /**
+     * This tag will be used to find the view while doing End to End testing.
+     */
+    void setViewTag();
```

Comment:
yeah, it doesn't need to be part of the interface. I was trying to implement something else.